remote/src/main/java/com/playone/mobile/remote/PlayoneRemoteImpl.kt

```diff
@@ -11,6 +11,7 @@ import com.playone.mobile.remote.model.PlayoneModel
 import com.playone.mobile.remote.model.UserModel
 import io.reactivex.Single
 
+@Suppress("NOTHING_TO_INLINE")
```

Comment:
Can you let me know any reason must to use this suppress?

Reply:
If they're called by function, it costs more than inline function.

Reply:
I am not so sure what you mean.. 

> costs more than inline function

I just feel confused because the warning says there is `performance impact`. What do you mean cost?

> I just reduce the writing for the developers.

the usage of inline function and normal function are the same, right? What you mean `reduce`?

Reply:
1. The keyword `inline` is the same as C's `inline`. That meaning just a few codes, it could be assembled to the code and the performance is also better than call other address of the function pointer. If I misunderstand, please let me know it.
2. I prefer to use the way of Functional Programming, I'd like to create some convenient functions as like `toXXX`. That's easier to understand what's that meaning and to code, isn't it?

Reply:
@pokk This is what I think, 
As we know `inline` function should be as small as possible, and its behavior is just as C's `inline` that is right. But in Kotlin, I think because there are some overhead of Kotlin lambda covert to Java bytecode(support 6, 7, 8)  so use `inline` we put the body of `lambda` into the calling place can avoid some overhead issues. That is what I think the warning reported and it is what the difference with `C` language. But for normal usage (without lambda parameters) and because the function body is very small It is better to just drop out that function, right?
Or if you want to reuse the code, I think you can make a normal function. 

However, I just think we should not easily put `Suppress` and ignore the warnings.

Reply:
@lekaha 

Okay, I totally agree with you this point. I could remove the annotation for the inline function warning.
I've not tested which one is slow or costly yet. I'll try to test it in the near future XD.